cli/cmd/vulnerabilty_exceptions.go

```diff
@@ -0,0 +1,354 @@
+			}),
+		),
+		tableFunc(func(t *tablewriter.Table) {
+			t.SetBorder(false)
+			t.SetAutoWrapText(false)
+		}),
+	)
+}
+
+func setProps(vuln api.VulnerabilityException) [][]string {
+	var (
+		details [][]string
+	)
+	details = append(details, []string{"DESCRIPTION", vuln.Props.Description})
+	details = append(details, []string{"UPDATED BY", vuln.Props.UpdatedBy})
+	details = append(details, []string{"LAST UPDATED", vuln.UpdatedTime})
+	details = append(details, []string{"CREATED", vuln.CreatedTime})
+	details = append(details, []string{"REASON", vuln.ExceptionReason})
+
+	if vuln.ExceptionType == api.VulnerabilityExceptionTypeHost.String() {
+		details = append(details, []string{"NAMESPACES", strings.Join(vuln.ResourceScope.Namespace, ", ")})
+		details = append(details, []string{"HOSTNAMES", strings.Join(vuln.ResourceScope.Hostname, ", ")})
+		details = append(details, []string{"EXTERNAL IPS", strings.Join(vuln.ResourceScope.ExternalIP, ",")})
+		details = append(details, []string{"CLUSTER NAMES", strings.Join(vuln.ResourceScope.ClusterName, ", ")})
+	} else if vuln.ExceptionType == api.VulnerabilityExceptionTypeContainer.String() {
+		details = append(details, []string{"NAMESPACES", strings.Join(vuln.ResourceScope.Namespace, ", ")})
+		details = append(details, []string{"IMAGE IDS", strings.Join(vuln.ResourceScope.ImageID, ", ")})
+		details = append(details, []string{"IMAGE TAGS", strings.Join(vuln.ResourceScope.ImageTag, ", ")})
+		details = append(details, []string{"REGISTRIES", strings.Join(vuln.ResourceScope.Registry, ", ")})
+		details = append(details, []string{"REPOSITORIES", strings.Join(vuln.ResourceScope.Repository, ", ")})
+	}
+
+	details = append(details, []string{"FIXABLE", vulnerabilityExceptionFixableEnabled(vuln.VulnerabilityCriteria.Fixable)})
+	details = append(details, []string{"CVES", strings.Join(vuln.VulnerabilityCriteria.Cve, ", ")})
+	details = append(details, []string{"SEVERITIES", strings.Join(vuln.VulnerabilityCriteria.Severity, ", ")})
+
+	if vuln.VulnerabilityCriteria.Package != nil {
+		packages, err := json.Marshal(vuln.VulnerabilityCriteria.Package)
+		if err != nil {
+			packages = []byte{}
+		}
+		details = append(details, []string{"PACKAGES", string(packages)})
+	} else {
+		details = append(details, []string{"PACKAGES", ""})
+	}
+	return details
+}
+
+func promptCreateVulnerabilityException() error {
+	var (
+		group  = ""
+		prompt = &survey.Select{
+			Message: "Choose a vulnerability exception type to create: ",
+			Options: []string{
+				"Host",
+				"Container",
+			},
+		}
+		err = survey.AskOne(prompt, &group)
+	)
+	if err != nil {
+		return err
+	}
+
+	switch group {
+	case "Host":
+		return createHostVulnerabilityException()
+	case "Container":
+		return createContainerVulnerabilityException()
+	default:
+		return errors.New("unknown vulnerability exception type")
+	}
+}
+
+func vulnerabilityExceptionFixableEnabled(fixable []int) string {
+	if fixable == nil {
```

Comment:
I think you also need to check if fixable has at least one element, if not, we will panic
```suggestion
	if fixable == nil || len(fixable) == 0 {
```